Kochbuch/src/app/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="userProfile">
+    <div class="col-md-6 well well-sm">
```

Comment:
Hier wird zum Beispiel die `well` Klasse verwendet, die in Bootstrap 4 durch die Card Komponente ersetzt wurde: https://getbootstrap.com/docs/4.0/migration/#components